lib/libteleirc.js

```diff
@@ -367,22 +396,34 @@ class TeleIrc {
                     // That is, if the config allows us to do that of course.
                     let emoji = msg.sticker.emoji;
                     if (emoji !== undefined && this.config.irc.sendStickerEmoji) {
-                        this.ircbot.say(this.config.irc.channel, this.config.irc.prefix + from + this.config.irc.suffix + " " + emoji);
+                        this.sendUserMessageToIrc(from, emoji);
                     }
                 } else {
                     // Check if this message contains a document/file to display in irc
                     if (msg.document) {
                         this.tgbot.getFileLink(msg.document.file_id).then((url) => {
-                            let newmessage = sg.document.file_name + " (" + msg.document.mime_type + ", " + filemsg.document.file_size_size + " bytes) : " + url;
-                            this.ircbot.say(this.config.irc.channel, this.config.irc.prefix + from + this.config.irc.suffix + " " + newmessage);
+                            let newmessage = msg.document.file_name + " (" + msg.document.mime_type + ", " + msg.document.file_size_size + " bytes) : " + url;
```

Comment:
Why change `sg` => `msg` and `filemsg` => `msg`?

Reply:
sg I think was a typo... it should have been msg all along.  If we tried uploading a document, teleirc crashed.  I have no idea where filemsg came from.. my guess is that was also a typo.